### PR TITLE
DTLS handshake text

### PR DESCRIPTION
**Pull request by mcr**
add text recommending a PMTU of 1024 bytes for handshake operations

### CODE REVIEW

**Comment by EskoDijk** [comment]
Good to mention this. We can add to the text the following:

A Pledge/EST-client on a constrained network MUST use the (D)TLS maximum fragment length extension ("max_fragment_length") defined in Section 4 of [RFC 6066] with the maximum fragment length set to a value of either 2^9 or 2^10.

The RFC 6066 reference is quite nice as it mentions constrained clients also. And it works! The registrar is thus signalled to use small fragments in its DTLS handshake messages, avoiding any delivery problems through Join Proxies.

**Review by EskoDijk — APPROVED** [review]
See Conversation for suggestion on RFC 6066 text and more detail on max frag length extension.

File: constrained-voucher.mkd [hunk 1]
```diff
@@ -486,6 +489,17 @@ If the Pledge included in its request an Accept Option for only the TBD287 ("app
...
+
+DTLS includes a mechanism to fragment the handshake messages.
+This is described in {{Section 4.4 of I-D.ietf-tls-dtls13}}.
+The protocol described in this document will often be used with a Join Proxy described in {{I-D.ietf-anima-constrained-join-proxy}}.
+The Join Proxy will need some overhead, while the maximum packet sized guaranteed on 802.15.4 networks is 1280 bytes.
+It is RECOMMENDED that a PMTU of 1024 bytes be assumed for the DTLS handshake.
+It is unlikely that any Packet Too Big indications {{RFC4443}} will be relayed by the Join Proxy.
+
+During the operation of the constrained BRSKI-EST protocol, the CoAP Blockwise transfer mechanism will be used when message sizes exceed the PMTU.
+
```
Comment (mcr) [inline constrained-voucher.mkd:502]:
```suggestion
A Pledge/EST-client on a constrained network MUST use the (D)TLS maximum fragment length extension ("max_fragment_length") defined in Section 4 of {{RFC6066}} with the maximum fragment length set to a value of either 2^9 or 2^10.

```

File: constrained-voucher.mkd [hunk 2]
```diff
@@ -78,6 +79,7 @@ normative:
 
 informative:
   RFC7228:
+  RFC4443:
```
Comment (mcr) [inline constrained-voucher.mkd:82]:
```suggestion
  RFC4443:
  RFC6066:
```

**Review by EskoDijk — APPROVED** [review]

**Pull request merged**